### ISSUE TITLE
[OTHER] Create a Basic Test for Playground Page

### ISSUE THREAD

**Issue by Vishrut19** [EddieHubCommunity/LinkFree#6019]
### What would you like to share?

The goal of this issue is to have a test file and test to navigate to the Playground page, and that checks the page loads correctly with the heading with no error.

**Comment by eddiejaoude** [comment]
There are test users you can use already `_test-profile-user-1` (`tests/profile.spec.js`)

**Comment by Vishrut19** [comment]
Okay but I would need its JSON file. Can you help me where I can find it?

**Comment by eddiejaoude** [comment]
Yes these example profiles are created on the test [setup](https://github.com/EddieHubCommunity/LinkFree/blob/main/tests/setup/global-setup.js), try running the tests and you will see https://github.com/EddieHubCommunity/LinkFree/blob/main/.gitignore#L32
